src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -257,7 +264,14 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, EvalVisitorCo
             {
                 if (FunctionImplementations.TryGetValue(func, out var ptr))
                 {
-                    result = await ptr(this, context.IncrementStackDepthCounter(childContext), node.IRContext, args);
+                    try
+                    {
+                        result = await ptr(this, context.IncrementStackDepthCounter(childContext), node.IRContext, args);

```

Comment:
>node.IRContext [](http://example.com/codeflow?start=99&length=14)

Catch knows the irContext, so we can apply here.

Reply:
update